src/main/kotlin/frc/robot/subsystems/arm/elevator/ElevatorIOSim.kt

```diff
@@ -0,0 +1,63 @@
+package frc.robot.subsystems.arm.elevator
+
+import com.ctre.phoenix6.controls.PositionVoltage
+import com.ctre.phoenix6.controls.VelocityVoltage
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.measure.Distance
+import edu.wpi.first.units.measure.LinearVelocity
+import edu.wpi.first.wpilibj.Timer
+import frc.robot.lib.motors.TalonFXSim
+import frc.robot.lib.motors.TalonType
+import frc.robot.lib.toAngle
+import frc.robot.lib.toAngular
+import frc.robot.lib.toDistance
+import frc.robot.lib.toLinear
+import org.littletonrobotics.junction.networktables.LoggedNetworkBoolean
+
+class ElevatorIOSim : ElevatorIO {
+    override val inputs: LoggedElevatorInputs = LoggedElevatorInputs()
+
+    private val isFloored = LoggedNetworkBoolean("isFloored", false)
+
+    private val velocityVoltageRequest: VelocityVoltage = VelocityVoltage(0.0)
+    private val positionVoltageRequest: PositionVoltage = PositionVoltage(0.0)
+
+    private val mainMotor =
+        TalonFXSim(2, GEAR_RATIO, 0.003, 1.0, TalonType.KRAKEN_FOC)
```

Comment:
If this is a single variable for both motors, this variable should just be called "motors".